src/Catel.Core/Catel.Core.Shared/Data/Extensions/IValidatableExtensions.cs

```diff
@@ -82,80 +80,6 @@ public static void AddFieldValidationResult(this IValidatable validatable, IFiel
         }
 
         /// <summary>
-        /// Gets the current business warnings.
-        /// </summary>
-        /// <returns>The warnings or <see cref="string.Empty"/> if no warning is available.</returns>
-        public static string GetBusinessRuleWarnings(this IValidatable validatable)
```

Comment:
We can't simply remove them since this is a breaking change. *if* we decide to remove them, we need to decorate them with ObsoleteEx.